osf/models/provider.py

```diff
@@ -406,17 +406,6 @@ def all_subjects(self):
             # TODO: Delet this when all PreprintProviders have a mapping
             return rules_to_subjects(self.subjects_acceptable)
 
-    @property
-    def has_highlighted_subjects(self):
-        return self.subjects.filter(highlighted=True).exists()
-
-    @property
-    def highlighted_subjects(self):
-        if self.has_highlighted_subjects:
-            return self.subjects.filter(highlighted=True).order_by('text')[:10]
-        else:
-            return sorted(self.top_level_subjects, key=lambda s: s.text)[:10]
-
```

Comment:
Why are these removable?

Reply:
They are redundant now they completely match the parent class (they were redundant before too, but now they really are).

Reply:
Gotcha. This sort of thing would be a good inclusion to the "Changes" section, to let the reviewer know that something not-as-obvious has changed and why.